themes/wilson_theme_starterkit/wilson_theme_starterkit.theme

```diff
@@ -341,6 +341,9 @@ function wilson_theme_starterkit_preprocess_breadcrumb(&$variables) {
  * document head.
  */
 function _wilson_theme_starterkit_background_styles($imageUri, $elementId, $styleNamePrefix) {
+  // Load file url generator service.
+  $url_generator = \Drupal::service('file_url_generator');
```

Comment:
Realising this is just fixing the error below, a learning point: if you're in a file that isn't namespaced (doesn't include a `namespace Drupal\...` line at the top of it), you don't have to add a backslash when calling Drupal static methods like this. 

`$url_generator = Drupal::service...` works fine :) 

The backslash basically says, "Don't look within the current namespace, start from the root". You're not in a namespace so all lookups will be from root, unless you've added a `use` statement.

Reply:
Thats really handy to know, thanks @iamsophiesk.